### PR TITLE
core(fr): convert tags-blocking-first-paint gatherer

### PR DESCRIPTION
**Pull request by adamraine**
Ref #11313

### CODE REVIEW

File: lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js [hunk 1]
```diff
@@ -82,7 +83,7 @@ async function collectTagsThatBlockFirstPaint() {
         };
       });
 
-    /** @type {Array<ScriptTag>} */
+    /** @type {ScriptTag[]} */
```
Comment (connorjclark) [inline lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js:86]: one step at a time... 🎉

**Review by connorjclark — APPROVED** [review]

File: lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js:86]: I have to agree. I mean changing them when you're already rewriting a block of code is one thing, but these are just completely untouched defs that are being changed here exclusively for preference.

If it's a war you want...

Comment (adamraine) [reply lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js:86]: In my defense I had a function `_getArtifact` which used `[]` and wanted the rest of the file to match. I ended up removing the function so I'm fine reverting.

In the future, should I just adhere to whatever preference is already in the file?

Comment (patrickhulce) [reply lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js]: > In the future, should I just adhere to whatever preference is already in the file?

Probably worth discussing in eng sync to make sure we all understand the same thing, but _my understanding_ of where we landed was use whichever you prefer in new code you (re)write, leave all others untouched and unmentioned in review.

Totally no malice interpreted from this btw, consistency is a great reason to clean up code that you find, it's just that we've already abandoned the consistency argument for these 😉

**Pull request merged**